### PR TITLE
xsimd 6.1.5 (new formula)

### PR DESCRIPTION
**Pull request by ktnyt**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/xsimd.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class Xsimd < Formula
+  desc "Modern, portable C++ wrappers for SIMD intrinsics"
+  homepage "https://xsimd.readthedocs.io/en/latest/"
+  url "https://github.com/QuantStack/xsimd/archive/6.1.5.tar.gz"
+  sha256 "ecc781520f7b2bfdeb51d1385683694f2e89be1a05b77b222b9783cfeb645034"
```
Comment (lembacon) [inline Formula/xsimd.rb]: Need a blank line after the `sha256` block.

Comment (ktnyt) [reply Formula/xsimd.rb]: Thank you for the feedback. I've fixed the issue but am unsure whether or not to re-submit a new PR to adhere to the "One formula per commit; one commit per formula" rule.

Comment (lembacon) [reply Formula/xsimd.rb]: That's fine, we can always squash them into one commit later.

File: Formula/xsimd.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args, "-DBUILD_TESTS=OFF"
```
Comment (DomT4) [inline Formula/xsimd.rb]: Would prefer:
```ruby
args = std_cmake_args
args << "-DBUILD_TESTS=OFF"

system "cmake", ".", *args
```
personally.

File: Formula/xsimd.rb (hunk 1 above)
Comment (ktnyt) [reply Formula/xsimd.rb]: OK that's nice to hear.

File: Formula/xsimd.rb (hunk 2 above)
Comment (ktnyt) [reply Formula/xsimd.rb]: I've patched the formula accordingly but used `cmake_args` instead of `args` just to be a little more verbose.

Comment (DomT4) [reply Formula/xsimd.rb]: Let's stick to `args` for consistency.

Comment (ktnyt) [reply Formula/xsimd.rb]: I'm very sorry for not noticing that `args` was _de facto_. Modifications were made accordingly.

Comment (DomT4) [reply Formula/xsimd.rb]: No worries! I appreciate it's a small thing for me/us to be picky about.

Comment (ktnyt) [reply Formula/xsimd.rb]: If my couple of seconds makes the life of everyone else easier I'm more than happy to comply =)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/2af2783ae4c092340eade9695ff4f19295c28392. Thank you for your first contribution to Homebrew @ktnyt; we appreciate it! Hopefully we've managed not to scare you away from future contributions 😸

**Pull request closed without merging**